scripts/prompt-release.js

```diff
@@ -39,21 +38,7 @@ const childProcess = require('@lerna/child-process')
       './packages/rum/src/apm-base.js'
     ])
 
-    /**
-     * Prompt for working directory clean
-     */
-    const answer = await PromptUtils.confirm(
```

Comment:
any reason why we delete this prompt since we are passing `--yes` anyways from the publish command.

Reply:
I thought it might not work, I'll keep as used to be then.

Reply:
I just tested it and it seems the `--yes` flag is not populated:

```
 npm run version --yes

> test-2fa@1.7.0 version /Users/vmartinez/work/poc/test-2fa
> node scripts/prompt-release

? Are these changes ok (Check your working directory before continuing) (ynH) 
```

```
 node scripts/prompt-release --yes
? Are these changes ok (Check your working directory before continuing) (ynH) 

```

Any ideas how to bypass that particular validation?

Reply:
We can remove this as we can generate `changelog.ascii` later and update the release notes in separate PR